app/src/androidTest/java/org/mozilla/focus/activity/AccessAboutAndYourRightsPagesTest.java

```diff
@@ -73,7 +79,7 @@ public void tearDown() {
     }
 
     @Test
-    public void AccessAboutAndYourRightsPagesTest() {
+    public void AboutAndYourRightsPagesTest() {
```

Comment:
\[PMD\] INFO: The method 'AboutAndYourRightsPagesTest\(\)' has a NCSS line count of 23\.